website/docs/reference/sinks/elasticsearch.md

```diff
@@ -74,6 +74,7 @@ endpoint][urls.elasticsearch_bulk].
   host = "http://10.24.32.122:9000" # optional, no default
   id_key = "id" # optional, no default
   index = "vector-%F" # optional, default
+  pipeline = "pipeline-name" # optional, no default
```

Comment:
Note that this file is automatically generated. This change should be accomplished by adding the option to `.meta/sinks/elasticsearch.toml.erb` and running `make generate`.

Reply:
pardòn. pr updated